app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -50,10 +50,12 @@ export default class SingleAddon extends React.Component{
         <td>
           <div className="addon-name">
             {
+              !app.install > 0 ?
```

Comment:
`app.install` is a boolean, why is it being compared to 0?

Reply:
Thanks for the feedback, i have fixed it